external-crates/move/crates/move-analyzer/src/analysis/typing_analysis.rs

```diff
@@ -1022,13 +1058,15 @@ impl<'a> TypingVisitorContext for TypingAnalysisContext<'a> {
                 }
                 TE::PackVariant(mident, name, vname, tyargs, fields) => {
                     visitor.add_datatype_use_def(mident, name);
+                    visitor.add_variant_use_def(mident, name, vname);
```

Comment:
This is a fix to show on hover for variants when the user inspects the `pack` operation